Makefile

```diff
@@ -122,19 +123,22 @@ normalized-link-check:
 .PHONY: markdown-link-check
 markdown-link-check: normalized-link-check
 	$(DOCKER_RUN) --rm \
+	    $(DOCKER_USER_IS_HOST_USER_ARG) \
 		--mount 'type=bind,source=$(PWD),target=/home/repo' \
-		lycheeverse/lychee \
+		$(LYCHEE_CONTAINER) \
 		--config home/repo/.lychee.toml \
 		--root-dir /home/repo \
 		--verbose \
+		--timeout=60 \
```

Comment:
thanks, we just started seeing timeout errors recently, which maybe aligns with this release: https://github.com/lycheeverse/lychee-action/releases/tag/v2.4.0, though I can't see anything in the change log for https://github.com/lycheeverse/lychee/releases/tag/lychee-v0.18.1 to explain it

I'm wondering if it the retry isn't occurring after timeout, but this is a good idea to increase from the default 20s to see if that helps 👍

Reply:
I'm going to re-run the CI check a few times on your PR to see if we get consistently better results now...

Reply:
Thanks. Good to know that this is likely unrelated to this change. I was wondering if it was related to GNU throttling traffic from GitHub (rather than anything to do with the lychee dependency).